assets/js/modules/analytics-4/datastore/audiences.js

```diff
@@ -181,8 +180,12 @@ const baseActions = {
 	 * If the `googlesitekit_post_type` custom dimension doesn't exist, creates it.
 	 *
 	 * @since 1.128.0
+	 * @since n.e.x.t Added `failedSiteKitAudienceResourceNames` parameter to retry failed Site Kit audience creation.
+	 *
+	 * @param {Array} failedSiteKitAudienceResourceNames List of failed Site Kit audience resource names to retry.
+	 * @return {Object} Object with `failedSiteKitAudienceResourceNames` and `error`.
 	 */
-	*enableAudienceGroup() {
+	*enableAudienceGroup( failedSiteKitAudienceResourceNames ) {
```

Comment:
The parameter name is incorrect as the values are audience slugs, not their resource names.

```suggestion
	*enableAudienceGroup( failedSiteKitAudienceSlugs ) {
```

Reply:
Updated.